src/index.html

```diff
@@ -16,17 +16,28 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="big_input_search form"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="small"
+      action="#"
+    >
+      <input
+        class="small_input_search  form"
```

Comment:
Avoid using spaces in class names. Use a hyphen or underscore to separate words instead of a space.